keps/sig-network/2086-service-internal-traffic-policy/kep.yaml

```diff
@@ -29,7 +29,6 @@ latest-milestone: "v1.22"
 milestone:
   alpha: "v1.21"
   beta: "v1.22"
-  stable: "v1.23"
```

Comment:
My new understanding now is that we don't update milestones until after we've actually landed the code for that milestone (as opposed to articulating what the desired milestone is)